### PR TITLE
AMB-907 - Pointing to int-dev docker container 

### PR DESCRIPTION
**Pull request by urchy**
## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by Valentinolucas — APPROVED** [review]

File: proxies/sandbox/apiproxy/targets/sandbox.xml [hunk 1]
```diff
@@ -15,6 +15,12 @@
         </Response>
     </PreFlow>
     <HTTPTargetConnection>
-      {{ HOSTED_TARGET_CONNECTION }}
+      <SSLInfo>
```
Comment (artronics) [inline proxies/sandbox/apiproxy/targets/sandbox.xml:18]: We can delete the entire sandbox directory.

**Comment by artronics** [comment]
@urchy there is no need for `sandbox` any more. You can delete both sandbox apigee and the docker container directory.

File: Makefile [hunk 2]
```diff
@@ -47,7 +47,7 @@ _dist_include="pytest.ini poetry.lock poetry.toml pyproject.toml Makefile build/
 release: clean copy-examples publish build-proxy
 	mkdir -p dist
 	for f in $(_dist_include); do cp -r $$f dist; done
-	cp ecs-proxies-deploy.yml dist/ecs-deploy-internal-dev.yml
+	#cp ecs-proxies-deploy.yml dist/ecs-deploy-internal-dev.yml
```
[the remark this answers is not in the document]
Comment (urchy) [reply Makefile:50]: I tried to delete the release but something within the pipeline needs it. 
So I end up leaving it there but without any action.
Do you know how can we change the pipeline to ignore this step?

**Review by artronics — APPROVED** [review]

**Pull request merged**